### DISCUSSION TITLE
Where are funcargs adjusted?

### DISCUSSION THREAD

**Discussion by Tusenka** [pytest-dev/pytest#13005] [Q&A]
here we use those:" https://github.com/pytest-dev/pytest/blob/acf130311806cb1a627ea0e21369c7d6f586663a/src/_pytest/python.py#L152
And here we store them as an empty list
https://github.com/pytest-dev/pytest/blob/acf130311806cb1a627ea0e21369c7d6f586663a/src/_pytest/python.py#L1587

**Comment by The-Compiler** [comment]
https://github.com/pytest-dev/pytest/blob/acf130311806cb1a627ea0e21369c7d6f586663a/src/_pytest/fixtures.py#L694-L698

(also, `{}` is an empty dictionary, not a list)
